radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/modules/ReleasenetForksModule.java

```diff
@@ -155,4 +157,45 @@ ForkBuilder releasenetV2() {
             100, // 100 max validators
             MSG.maxLength()));
   }
+
+  @ProvidesIntoSet
+  ForkBuilder olympiaRadixProtocolFork1() {
+    return new ForkBuilder(
```

Comment:
Can't we avoid duplicating this code?

Reply:
Sure - I was following precedent, but it's probably a good idea.

It's probably best not to do this when we're merging a release branch into main; would you be happy if we remembered to do this at our next fork creation PR?

Also; apologies for deleting your comment on the last PR with this - the fork had to go out ASAP, and that PR was being shared publicly to explain the fork, so I wanted to avoid confusing the public with code quality comments :D

Reply:
Sure, no worries ;)